RecoTauTag/HLTProducers/src/L1HLTTauMatching.cc

```diff
@@ -1,9 +1,13 @@
 #include "RecoTauTag/HLTProducers/interface/L1HLTTauMatching.h"
 #include "Math/GenVector/VectorUtil.h"
+#include "DataFormats/Common/interface/Handle.h"
 #include "DataFormats/L1Trigger/interface/L1JetParticle.h"
 #include "DataFormats/L1Trigger/interface/L1JetParticleFwd.h"
 #include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
 #include "FWCore/Utilities/interface/EDMException.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "DataFormats/JetReco/interface/CaloJetCollection.h"
```

Comment:
I can't see why this is needed.
```suggestion
```

Reply:
Indeed, it is not needed.
I removed a few of them from the files that I touched, I forgot to remove also this one.
Since I am on it, let clean also this away